app/views/campaigns/overview.html.haml

```diff
@@ -1 +1,31 @@
 = render 'nav'
+
+.container.campaign_main
+  %section.overview.container
+    .stat-display
+      = render 'courses/header_stats', presenter: @presenter
+    .primary
+      = form_for(@campaign, url: campaign_path(@campaign.slug), html: { class: 'module campaign-description editable' }) do
+        = hidden_field(:campaign, :slug)
+        .section-header
+          %h3
+            = succeed ':' do
+              = t('campaign.campaign')
+            = @campaign.title
+          .controls
+            - if current_user&.admin?
```

Comment:
instead of checking current_user at each point in the view, maybe this makes more sense to set once in the controller as `@editable`. That way it will be easy to patch in the `campaigns_users` permissions when we get there.